src/Config/CS.props

```diff
@@ -1,14 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-   <ImportGroup Label="PropertySheets" Condition=" Exists('$(SolutionDir)Config/user_local.props') ">
-    <Import Project="$(SolutionDir)Config/user_local.props" />
-  </ImportGroup>
+<Project>
   <PropertyGroup>
-	<DynamoCorePath Condition = "!Exists('$(DynamoCorePath)')">$(SolutionDir)\packages\DynamoVisualProgramming.Core.1.0.0-beta3\lib\net45</DynamoCorePath>
-	<DynamoZeroTouchPath Condition = "!Exists('$(DynamoZeroTouchPath)')">$(SolutionDir)\packages\DynamoVisualProgramming.ZeroTouchLibrary.1.0.0-beta3\lib\net45</DynamoZeroTouchPath>
-	<DynamoWpfUIPath Condition = "!Exists('$(DynamoWpfUIPath)')">$(SolutionDir)\packages\DynamoVisualProgramming.WpfUILibrary.1.0.0-beta4\lib\net45</DynamoWpfUIPath>
-	<DynamoWpfUINodesPath Condition = "!Exists('$(DynamoWpfUINodesPath)')">$(DynamoWpfUIPath)</DynamoWpfUINodesPath>
-	<DynamoServicesPath Condition = "!Exists('$(DynamoServicesPath)')">$(SolutionDir)\packages\DynamoVisualProgramming.DynamoServices.1.0.0-beta3\lib\net45</DynamoServicesPath>
-	<DynamoTestPackagePath Condition = "!Exists('$(DynamoTestPackagePath)')">$(SolutionDir)\packages\DynamoVisualProgramming.Tests.1.0.0-beta4\lib\net45</DynamoTestPackagePath>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <PlatformTarget >x64</PlatformTarget>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
+    <TargetFramework>net8.0</TargetFramework>
+    <FileAlignment>512</FileAlignment>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
is this needed for all projects? I guess it's not that important now but it would be nice if our ZT and extension samples could run on linux builds.

Reply:
removed